src/site/content/en/blog/file-system-access/index.md

```diff
@@ -35,6 +35,13 @@ read or save changes directly to files and folders on the user's device. Beyond
 files, the File System Access API provides the ability to open a directory and enumerate its
 contents.
 
+{% Aside %}
+The File System Access API—despite the similar name—is distinct from the
+[`FileSystem`](https://developer.mozilla.org/en-US/docs/Web/API/FileSystem) interface exposed by the
+[File and Directory Entries API](https://wicg.github.io/entries-api/#api-domfilesystem) and the deprecated
+[File API: Directories and System](https://www.w3.org/TR/file-system-api/) specification.
+{% endAside %}
+
```

Comment:
Maybe explain why it's different than these other APIs. If you do that, give it a heading instead of putting it in an aside.

Reply:
I'd rather not add more information, just point people at links. The article is already very long.

Reply:
I have to insist on this one. We at least need a sentence that summarizes how this API is different, especially since I need to read three different pages to figure that out for myself.